doc/nrf/ug_bootloader_adding.rst

```diff
@@ -104,20 +114,18 @@ Therefore, if a directory in ``/home/user/ncs`` contains signing keys, say ``_ke
 
          -DCONFIG_SB_SIGNING_KEY_FILE=\"/home/user/ncs/_keys/priv.pem\"
 
-      Or, if an environment variable ``NCS`` is set to ``/home/user/ncs``,
+      Or, if you set an environment variable named ``NCS`` to ``/home/user/ncs``,
 
       .. code-block:: console
 
          -DCONFIG_SB_SIGNING_KEY_FILE=\"$NCS/_keys/priv.pem\"
 
-On Unix systems, you can also symbolically link the file into the sample directory, though it will be seen by git as an untracked file.
+On Unix systems, you can also symbolically link the file into the sample directory, even if git will see it as an untracked file.
```

Comment:
I'm thinking that we should just remove this sentence. This suggests that symbolically linking is a valid alternative, but git will see them as an untracked file and pollute the untracked file list with a new "file". If it gets committed by accident and cloned onto a Windows system, the link won't work the same as in Unix.

If users want to symbolically link files within their own fork, that should be their own prerogative but not an official suggestion from NCS.

Thoughts?

Reply:
fixed